src/Main.java

```diff
@@ -1,5 +1,117 @@
 public class Main {
     public static void main(String[] args) {
+        System.out.println("Задача 1");
+        short clientOS = 1;
+        if (clientOS == 0) {
+            System.out.println("Установите версию для IOS по ссылке");
+        } else {
+            System.out.println("Установите версию приложения для Android по ссылке");
+        }
+        System.out.println("Задача 1 Вариант 2");
+        char clientOS2 = 'i';
+        switch (clientOS2) {
+            case 'i':
+                System.out.println("Установите версию приложения для IOS по ссылке");
+                break;
+            case 'a':
+                System.out.println("Установите версию приложения для Android по ссылке");
+                break;
+        }
+        System.out.println("Задача 2");
+        int clientDeviceYear = 2014;
+        short OS = 1;
```

Comment:
```suggestion
        short os = 1;
```
названия переменных с маленькой буквы